cli/epicli.py

```diff
@@ -253,6 +253,7 @@ def upgrade_parser(subparsers):
         'ignite',
         'image_registry',
         'kafka',
+        'kafka-exporter',
```

Comment:
Minor comment - this is the only place where you are using dash instead of underscore (other components in cli are using underscores)

Reply:
I would suggest to keep it consistent and use underscore.

Reply:
Changed